project_name/settings/base.py

```diff
@@ -95,10 +100,15 @@ def base_dir_join(*args):
 # Webpack
 WEBPACK_LOADER = {
     'DEFAULT': {
-        'CACHE': True,
+        'CACHE': False,
         'BUNDLE_DIR_NAME': 'bundles/',  # must end with slash
         'STATS_FILE': base_dir_join('webpack-stats.json'),
         'POLL_INTERVAL': 0.1,
         'IGNORE': ['.+\.hot-update.js', '.+\.map']
     }
 }
+
+# Celery
+CELERY_ACCEPT_CONTENT = ['json']
+CELERY_TASK_SERIALIZER = 'json'
+CELERY_RESULT_SERIALIZER = 'json'
```

Comment:
Avoids security problems, pickle is deprecated. But reduces flexibility. All task params must be JSON serializable